assets/app/views/edit/build-config.html

```diff
@@ -255,7 +255,13 @@
 
 
                             <div class="multiple-image-source" ng-if="sourceImages.length !== 1">
-                              <label for="imageSourceFrom">Image Source From</label>
+                              <label for="imageSourceFrom">Image Source From<span class="help action-inline">
+                                <a href>
+                                  <i class="pficon pficon-info" aria-hidden="true"
+                                  data-toggle="tooltip" data-original-title="Build Config contains more then one Image Source. To edit them use the YAML editor.">
```

Comment:
Minor nit on the wording:

"This Build Config contains more then one Image Source. To edit them, use the YAML editor."